app/models/election.js

```diff
@@ -55,7 +55,8 @@ const electionSchema = new Schema({
   },
   accessCode: {
     type: Number,
-    default: Math.floor(Math.random() * (10000 - 1000) + 1000),
+    // https://mongoosejs.com/docs/defaults.html#default-functions
+    default: () => Math.floor(Math.random() * (10000 - 1000) + 1000),
```

Comment:
Why `10000 - 1000`?

Reply:
I don't codes under 1000, This will give codes between 1000 and 9999.

Reply:
Maybe it's better to just write 9000 in the middle, but I wanted people to get what that is was 1000->9999

Reply:
fixed it